src/eventDispatcher.js

```diff
@@ -0,0 +1,21 @@
+function EventDispatcher() {
+    this.eventDispatcherMap = {};
+};
+
+EventDispatcher.prototype.bind = function(eventName, handler) {
```

Comment:
Similar to bind, we'll need several other functions
1) `unbind(eventHame, handler)`
2) `unbindAll()` - removes all listeners for every event name

Please add them.